src/steth/scWETH.sol

```diff
@@ -140,7 +140,7 @@ contract scWETH is sc4626, IFlashLoanRecipient {
 
         // store the old total
         uint256 oldTotalInvested = totalInvested;
-        uint256 assets = totalAssets();
+        uint256 assets = getCollateral() - getDebt();
```

Comment:
_rebalancePosition() deposits all float into the strategy. Thus totalAssets() == totalCollateral()-getDebt() after the _rebalancePosition() call on line 139 since asset.balanceOf(address(this))==zero after line 139. So I don't under how this changes the code in any way.

Reply:
oh, good find. then I guess the finding is incorrect as originally thought. will close this PR

Reply:
yes agree.